spec/multiple_cursors_spec.rb

```diff
@@ -170,47 +170,6 @@ def type(string)
 
 end
 
-describe "Multiple Cursors when using insert mapings" do
```

Comment:
why were these tests removed?

Reply:
Because the absence of "sleep" induced a 100%CPU usage, breaking those tests. I just reintroduced them now that I fixed the sleep issue.